install-pip-dependencies.sh

```diff
@@ -2,22 +2,32 @@
 
 set -o nounset -o errexit
 
-
+requirements_file=$1
+install_location=$2
+
+if [ -z "$install_location" ]
```

Comment:
```suggestion
if [[ -z "$install_location" ]]
```

Reply:
Out of curiosity, what do those extra brackets *do*?